Cabal/src/Distribution/PackageDescription/Check.hs

```diff
@@ -669,6 +670,9 @@ ppExplanation (UnknownArch unknownArches) =
     "Unknown architecture name " ++ commaSep (map quote unknownArches)
 ppExplanation (UnknownCompiler unknownImpls) =
     "Unknown compiler name " ++ commaSep (map quote unknownImpls)
+ppExplanation (MissingUpperBounds pname) =
+  "'" ++ unPackageName pname ++ "' misses upper bounds, add them"
```

Comment:
You can probably the convenience `quote` instead of "'" ++ xyz "'"

Reply:
Also why not indenting four spaces, since that is what is been used in `ppExplanation`.